src/wanderer/eightbit.ts

```diff
@@ -0,0 +1,70 @@
+import { Location, numericModifier } from "kolmafia";
+import { DraggableFight } from ".";
+import { WandererTarget } from "./lib";
+import { $item, $location, clamp, get, have, NumericModifier } from "libram";
+import { garboValue } from "../value";
+import { globalOptions } from "../config";
+
+const FAT_LOOT_TOKEN_COST = 20000;
+
+export const bonusColor = ["black", "blue", "green", "red"] as const;
+export type BonusColor = (typeof bonusColor)[number];
+
+// taken from tour guide, who took it from beldur
+
+const locationColor: Record<BonusColor, Location> = {
+  black: $location`Vanya's Castle`,
+  blue: $location`Megalo-City`,
+  green: $location`Hero's Field`,
+  red: $location`The Fungus Plains`,
+};
+
+const minimumForPoints: Record<BonusColor, number> = {
+  black: 300,
+  red: 150,
+  blue: 300,
+  green: 100,
+};
+
+const modifierForColor: Record<BonusColor, NumericModifier> = {
+  black: "Initiative",
+  red: "Meat Drop",
+  blue: "Damage Absorption",
+  green: "Item Drop",
+};
+
+function pointsPerTurn(color: BonusColor) {
+  const current = get("8BitColor") === color;
+  const addedBonus = current ? 100 : 50;
+  const denominator = current ? 20 : 10;
+  const rawPoints = clamp(
+    numericModifier(modifierForColor[color]) - minimumForPoints[color],
+    0,
+    300,
+  );
```

Comment:
There's a mafia

Reply:
what is it

Reply:
number [eightBitPoints](https://wiki.kolmafia.us/index.php?title=eight_bit_points)(Location)
number [eightBitPoints](https://wiki.kolmafia.us/index.php?title=eight_bit_points)(Location, string, number)